tox.ini

```diff
@@ -13,7 +13,11 @@ deps =
     molecule_{lint,syntax,test}: docker
     molecule_{lint,syntax,test}: jmespath
     molecule_{lint,syntax,test}: molecule
-    molecule_{lint,syntax,test}: selinux
+    # The selinux pypi shim does not work with Ubuntu (as used by Travis), yet.
+    # Therefore use a fork with Ubuntu support. This can be changed once the
+    # update is available on PyPi.
+    # molecule_{lint,syntax,test}: selinux
+    molecule_{lint,syntax,test}: git+https://github.com/tyll/selinux-pypi-shim@fulllocation
```

Comment:
@i386x @vcrhonek I think we can revert this part given that https://github.com/pycontribs/selinux/pull/3 is merged and released.